modules/javafx.media/src/main/native/gstreamer/gstreamer-lite/gstreamer/gst/gst.c

```diff
@@ -122,6 +122,11 @@
 
 #include "gst.h"
 
+#ifdef STATIC_BUILD
```

Comment:
I think this should also be qualified by `GSTREAMER_LITE` to indicate that it is a JavaFX-specific modification.

Reply:
done.